prefect_aws/credentials.py

```diff
@@ -67,3 +67,61 @@ def get_boto3_session(self) -> boto3.Session:
             profile_name=self.profile_name,
             region_name=self.region_name,
         )
+
+
+class MinIOCredentials(Block):
+    """
+    Block used to manage authentication with MinIO. Refer to the
+    [MinIO docs](https://docs.min.io/docs/minio-server-configuration-guide.html)
+    for more info about the possible credential configurations.
+
+    Args:
+        minio_root_user: Admin or root user
+        minio_root_password: Admin or root password
+
+    Example:
+        Load stored MinIO credentials:
+        ```python
+        from prefect_aws import MinIOCredentials
+
+        minio_credentials_block = MinIOCredentials.load("BLOCK_NAME")
+        ```
+    """  # noqa E501
+
+    # placeholder logo
+    _logo_url = "https://www.outsystems.com/Forge_CW/_image.aspx/Q8LvY--6WakOw9afDCuuGRJMfoXxz3o9m-m-TCt8U4M=/minio-client-2022-06-15%2011-46-17"  # noqa
```

Comment:
Can you find an image that has a transparent background? I can upload the image to our CMS and update the URL here if you find one.

Reply:
Great, done